### PR TITLE
Add Cory Snider (corhere) as maintainer

### PR DESCRIPTION
**Pull request by thaJeztah**
Cory has actively participated in the project for many months, assisted in several
security advisories, code review, and triage, and (in short) already acted a
maintainer for some time (thank you!).

I nominated Cory as a maintainer per e-mail, and we reached quorum, so opening
this pull request to (should he choose to accept it) be added as a maintainer.

@moby/moby-maintainers @corhere

**- A picture of a cute animal (not mandatory but encouraged)**

140924110551-klm-dog1-story-top

### CODE REVIEW

**Review by tonistiigi — APPROVED** [review]

**Review by corhere — APPROVED** [review]

**Review by tianon — APPROVED** [review]
As I put in that email thread, vigorously LGTM :smile:

I'm going to go ahead and merge this since we already had maintainer quorum and now we have @corhere approval too. :+1:

**Review by samuelkarp — COMMENTED** [review]
I'm here too late to click the approve button but LGTM!

**Comment by AkihiroSuda** [comment]
LGTM

**Pull request merged**
